lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -192,6 +193,18 @@ sub _link_asset {
     return $target->to_string;
 }
 
+sub _link_repo {
+    my ($source_dir, $pooldir, $target_name) = @_;
+    $pooldir = path($pooldir);
+    my $target = $pooldir->child($target_name);
+    unlink $target if -e $target;
+    return {error => "The source directory $source_dir does not exist"} unless -e $source_dir;
+    eval { symlink($source_dir, $target) or die qq{Cannot create symlink from "$source_dir" to "$target": $!} };
```

Comment:
Do we support systems without `symlink`? Otherwise that function should not be able to raise an exception. And you could just write:
```perl
return {error => qq{Cannot create symlink from "$source_dir" to "$target": $!}} unless symlink($source_dir, $target);
```